modules/data-entry/src/main/frontend/src/dataHomepage/FilterComponents/DateFilter.jsx

```diff
@@ -23,7 +23,7 @@ import PropTypes from "prop-types";
 
 import FilterComponentManager from "./FilterComponentManager.jsx";
 import { DEFAULT_COMPARATORS, UNARY_COMPARATORS, VALUE_COMPARATORS } from "./FilterComparators.jsx";
-import { DATE_FORMATS, MONTH_FORMAT } from "../../questionnaire/DateQuestion.jsx";
+import { DATE_FORMATS, MONTH_FORMAT, MONTH_FORMATS } from "../../questionnaire/DateQuestion.jsx";
```

Comment:
Shouldn't `MONTH_FORMAT` be removed?
```suggestion
import { DATE_FORMATS, MONTH_FORMATS } from "../../questionnaire/DateQuestion.jsx";
```